src/olympia/constants/base.py

```diff
@@ -162,6 +167,7 @@
     ADDON_LPAPP: 'language-tools',
     _ADDON_SEARCH: 'search-tools',
     ADDON_STATICTHEME: 'themes',
+    ADDON_SITE_PERMISSION: 'sitepermission',
```

Comment:
```suggestion
    ADDON_SITE_PERMISSION: 'site-permission',
```
? (and for `ADDON_TYPE_CHOICES_API`)

Reply:
This is my overly optimistic attempt to bring us closer to what Firefox uses (we're already diverging since they use `theme` for `statictheme`, and `langpack` and not `language`)

Reply:
I look forward to the future patch to update those in the v5 api 😉